llvm/test/Transforms/LoopVectorize/optsize.ll

```diff
@@ -340,7 +325,7 @@ for.cond.cleanup28:
 
 for.body29:
   %i24.0170 = phi i16 [ 0, %entry], [ %inc37, %for.body29]
-  %add33 = add i16 undef, %i24.0170
+  %add33 = add i16 3, %i24.0170
```

Comment:
Looks like this a negative test for a SCEV check, maybe plumb through an argument here just we get the same codepath as the test below?